src/pages/PostJobs/Form.tsx

```diff
@@ -0,0 +1,261 @@
+import React,{useState} from 'react';
+import { ErrorBoundary } from '../../components';
+import  {JobDet}  from '../../components/DataModels/JobDet';
+import './Form.scss';
+import JobDetails from '../Home/JobDetails';
+
+const Form = () => {
+    const [preview,setPreview] = useState("form show");
+    const [currentJobView,setCurrenetJobView]=useState(null);
+    const [qualification,setQualification] = useState([
+      {id:'Btech',qual:"BTech"},
+    ]);
+    const removeLi = (id:string)=>{
+      let ele=document.getElementById(id);
+      console.log("ele",ele);
+      // ele.parentNode.remove(ele);
+    }
+    const addQualification = (id:string)=>{
+      let wrong=document.createElement('span');
+      wrong.appendChild(document.createTextNode('x'));
+      let val= document.getElementById(id+"input") as HTMLInputElement | null;
+      var li = document.createElement('li');
+      li.setAttribute("id",val.value);
```

Comment:
you can maintain state for value and you can write a map in the return statement

Reply:
Annaya ala try chesina kani console lo print chesthey update aithunnai kani html form lo update avvatle